### PR TITLE
Graduate wheel-building code out of //experimental

### LINKED ISSUE #374: Ship /experimental directory in released versions
# 🚀 feature request

### Relevant Rules

Ship `/experimental` in releases

### Description

Currently we're using the SHA directly from git so that we can access the experimental rules. We would like to move to released versions instead.

### Describe the solution you'd like

Ship the `experimental` directory in released versions

### Describe alternatives you've considered

 Have you considered any alternative solutions or workarounds?

### PR DESCRIPTION
**Pull request by pstradomski**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Does not include precompiled binaries, eg. `.par` files. See [CONTRIBUTING.md](/CONTRIBUTING.md) for info
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe: Move wheel-building rules out of //experimental.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

No, the rules are still re-exported at the old location

FIXES #374

### CODE REVIEW

File: tools/BUILD [hunk 1]
```diff
@@ -11,10 +11,19 @@
...
+
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
 
+# Implementation detail of py_wheel rule.
+py_binary(
+    name = "wheelmaker",
+    srcs = ["wheelmaker.py"],
+    visibility = ["//visibility:public"],
```
Comment (thundergolfer) [inline tools/BUILD:24]: The package default is `//visibility:public` so maybe this is redundant.

**Review by thundergolfer — APPROVED** [review]
LGTM. 

Now that this is out of `experimental/`, `py_wheel` probably warrants a mention in the README. But that is not a big deal and can be sorted out in a followup.

File: tools/BUILD (hunk 1 above)
Comment (pstradomski) [reply tools/BUILD]: Removed.

**Comment by UebelAndre** [comment]
Could a new release be made for this change? Or what goes into making a new release?

**Pull request merged**